src/components/SingleArticle.jsx

```diff
@@ -17,26 +19,21 @@ export default function SingleArticle() {
   }, [article_id]);
 
   if (isLoading) return <p>Article is loading...</p>;
-  let dateStr = "";
-  if (article.created_at) {
-    const date = new Date(Date.parse(article.created_at));
-    const options = {
-      weekday: "long",
-      year: "numeric",
-      month: "long",
-      day: "numeric",
-    };
-    dateStr = "On " + date.toLocaleDateString(undefined, options);
-  }
+  const dateStr = toDateStr(article.created_at);
 
   return (
     <section>
       <article>
-        <p>Posted by {article.author}</p>
-        <p>{dateStr}</p>
-        <h2>{article.title}</h2>
+        <h2>
+          Posted by {article.author}
+          {dateStr}
+        </h2>
+        <h3>{article.title}</h3>
         <VoteCard votes={article.votes} article_id={article_id} />
-        <p>{article.comment_count} comments</p>
+        <HashLink smooth to={`/articles/${article_id}/comments#list-comments`}>
```

Comment:
To use this, you'll need to use the NavHashLink component rather than the HashLink component. The hash (#) points to an HTML ID on a page, which I don't think your comments have. Try giving them an ID and seeing what you can do with that. But this will be for after this week, as we have not asked for this functionality so it would be best to get on with other tickest and coming back to this later :)

Reply:
the comments have this tag. and it works, but if I am already on this page. But doesn't work as it is in my head should do.